### PR TITLE
[WALLET + REACT] Add Blocto Wallet

### PR DESCRIPTION
**Pull request by q20274982**
We can't wait to announce that we have added Blocto wallet support to thirdweb.

Blocto wallet is a smart contract wallet that aims to lower the entry barrier for users into Web3,
users can connect their wallets without the need to install browser extensions. Furthermore, we have recently added support for the ERC-4337 mode.

We understand the significance of this feature to our users, and we have committed ourselves to its implementation to enhance the usability of thirdweb and Blocto wallet.

Additional Information:
- Blocto SDK: https://github.com/portto/blocto-sdk
- Blocto Docs: https://docs.blocto.app/blocto-sdk/overview

image

### CODE REVIEW

**Comment by MananTank** [comment]
Hey @q20274982, Thanks for the PR, We are very excited to get this merged! 

Can you please enable "Allow edits by maintainers" in this PR so that we can push a few fixes in this branch?

**Comment by q20274982** [comment]
> Hey @q20274982, Thanks for the PR, We are very excited to get this merged!
> 
> Can you please enable "Allow edits by maintainers" in this PR so that we can push a few fixes in this branch?

Hey @MananTank , It seems that due to the PR being opened from the organization, I am unable to enable option "Allow edits by maintainers".  Could you please to use "Request Changes" as an alternative?

**Comment by q20274982** [comment]
Hey @MananTank , I checked the two failed jobs on GitHub Action[[1]](https://github.com/thirdweb-dev/js/actions/runs/5661814302/job/15340640750#step:5:882), [[2]](https://github.com/thirdweb-dev/js/actions/runs/5661814302/job/15340641244?pr=1361#step:6:343), and from the logs of the failures, it seems that they are not directly related to the code I added. Could you please help me check?

Also, regarding the error message in [Unit Tests at L415 command](https://github.com/thirdweb-dev/js/actions/runs/5661814302/job/15340640750?pr=1361#step:5:422) `(/home/runner/work/js/js/packages/sdk) pnpm run test:evm exited (1)`, I tried to reproduce it in my local environment, and it succeeded. Perhaps it's an error caused by the outdated GitHub Action cache?

**Comment by MananTank** [comment]
Hi @q20274982, Yes we are aware of this. Failed jobs are not related to this PR. 
I've reviewed the PR - LGTM, We'll merge this shortly

**Review by MananTank — APPROVED** [review]
LGTM

File: .changeset/fair-beers-deny.md
Comment (MananTank) [inline .changeset/fair-beers-deny.md:14]: Hey @q20274982, Can you update the example in changeset? `bloctoWallet()` does not take `chain` anymore

Comment (q20274982) [reply .changeset/fair-beers-deny.md]: sure, resolved in [chore: update changeset](https://github.com/thirdweb-dev/js/pull/1361/commits/ed7ad13576cd00b2933afc36937d69a35f64fae1)

**Comment by joaquim-verges** [comment]
Thank you for your contribution @q20274982 !

Blocto support will be rolled out with the next release 🎉

**Comment by q20274982** [comment]
Thanks for your time and support. I can't wait for the release! 🚀

**Pull request merged**
